### DISCUSSION TITLE
Can't open personal fork from my organisation if organisation reaches limit

### DISCUSSION THREAD

**Discussion by mserranom** [community/community#6517] [Codespaces]
Bug report:

1. I'm able to open codespaces in any repo in github outside my organisation
1. My organisation reaches spending limit in codespaces
1. I fork a repo from my organisation, to use my personal account for billing
1. I still can't open a codespace in my personal fork due to "organisation reached spending limit"

**Comment by tanmayeekamath** [comment]
Hey @mserranom, thanks for reaching out. We're investigating this, will get back to you with an update shortly.

**Comment by tanmayeekamath** [comment]
Hey @mserranom, can you please share your org and repo name as well as personal fork name? Feel free to send it to tanmayeekamath@github.com as well, and we can take it from there.

**Reply by mserranom** [reply]
Sure, it's overleaf/overleaf and mserranom/overleaf. Cheers!
